### PR TITLE
feat: #1 added a parameter to configure the number of examples shown …

### PR DESCRIPTION
**Pull request by msooklaris**
…for each verb

### CODE REVIEW

File: greek_verbs/src/anki_flashcard.py [hunk 1]
```diff
@@ -27,17 +27,20 @@ def conjugation_to_html(verb: str, conj: dict) -> str:
     return html_template
 
 
-def format_example_usages(conj: dict, example_usages: dict) -> str:
+def format_example_usages(conj: dict, example_usages: dict, num_examples) -> str:
```
Comment (tsouchlarakis) [inline greek_verbs/src/anki_flashcard.py]: needs typehint

File: greek_verbs/src/anki_flashcard.py [hunk 2]
```diff
@@ -27,17 +27,20 @@ def conjugation_to_html(verb: str, conj: dict) -> str:
...
     """
     Extract example usages from the conjugation and format them for
     an Anki flashcard. The input is a dictionary with key:value pairs
     of Greek:English for each example usage.
     """
     if len(example_usages):
+
```
Comment (tsouchlarakis) [inline greek_verbs/src/anki_flashcard.py]: this is a great change

File: greek_verbs/src/anki_flashcard.py [hunk 3]
```diff
@@ -27,17 +27,20 @@ def conjugation_to_html(verb: str, conj: dict) -> str:
...
         template = '{greek}<br><em>{english}</em>'
 
         usages = []
-        for greek, english in example_usages.items():
+
+        for i, (greek, english) in enumerate(example_usages.items()):
```
Comment (tsouchlarakis) [inline greek_verbs/src/anki_flashcard.py:40]: `i` will start 0, so when `i == num_examples` there will be `num_examples+1` usages appended

File: greek_verbs/src/show.py [hunk 4]
```diff
@@ -94,12 +94,13 @@ def print_imperative_table(conjugation: dict) -> str:
...
 
-def print_example_usages(example_usages: dict) -> None:
+def print_example_usages(example_usages: dict, num_examples: int) -> None:
     """
     Print example usages if present.
     """
     if len(example_usages):
-        for greek, english in example_usages.items():
+        for i, (greek, english) in enumerate(example_usages.items()):
+            if i == num_examples: break
```
Comment (tsouchlarakis) [inline greek_verbs/src/show.py:103]: Same issue as on line 42

**Review by tsouchlarakis — CHANGES_REQUESTED** [review]
Requested a couple changes

File: greek_verbs/src/anki_flashcard.py (hunk 3 above)
Comment (msooklaris) [reply greek_verbs/src/anki_flashcard.py:40]: no there won't be. I tested it

**Review by tsouchlarakis — APPROVED** [review]
LGTM

**Pull request merged**